### PR TITLE
Status percentage

### PR DESCRIPTION
**Pull request by luizdanielmlima**
## Description

Added StatusPercentage component

## Motivation and Context

Add a new component to library

## How Has This Been Tested?

Tested on Storybook

### Desktop
image

## Types of changes

- [ ] Fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Build (Changes that affect the build system or dependencies)
- [ ] Chore (Other changes that do not modify src or test files)
- [ ] CI (Changes to our CI configration and scripts)
- [ ] Docs (Changes to documentation files)
- [ ] Style (Changes to coding style, like linters and prettier)
- [ ] Refactor (Changes that do not add new functionality or alter current ones)
- [ ] Performance (Performance improvements)
- [ ] Test (Adding or changing tests or the testing setup)
- [ ] Revert (Reverting a previous change)

## Checklist:

- [ ] I have made sure my changes work properly on iOS devices running Safari.
- [ ] I have made sure my changes work properly on Android devices running Chrome.
- [x] I have made sure my changes work properly on Desktop.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://49--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://49--logrock-pebbles.netlify.app&speed=1

File: lib/components/StatusPercentage/StatusPercentage.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  radius: number;
+  stroke: number;
+}
+
+const StatusPercentage: FC<StatusPercentageProps> = ({
+  percentage,
+  size = "medium",
+  children,
+}) => {
+  const getRingSize = (): StatusPercentageSizeType => {
```
Comment (cld-santos) [inline lib/components/StatusPercentage/StatusPercentage.tsx]: this method is exposing a static data structure, why not have a data structure instead, like:
```
{
  small: { radius: 18, stroke: 4}
  medium: { radius: 18, stroke: 4}
}
``` 
?

File: lib/components/StatusPercentage/StatusPercentage.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      case "small":
+        return { radius: 18, stroke: 4 };
+      case "medium":
+        return { radius: 20, stroke: 4 };
+      case "large":
+        return { radius: 24, stroke: 5 };
+    }
+  };
+
+  const getTextSize = (): string => {
```
Comment (cld-santos) [inline lib/components/StatusPercentage/StatusPercentage.tsx]: same here, why a method when a data structure is more suitable.

File: lib/components/StatusPercentage/StatusPercentage.tsx (hunk 1 above)
Comment (luizdanielmlima) [reply lib/components/StatusPercentage/StatusPercentage.tsx]: Good tip, thanks, I will change the solution ;-)

File: lib/components/StatusPercentage/StatusPercentage.styled.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+export const StatusPctWrapper = styled.div<{ fontSize: string }>`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+
+  font-family: Archivo, sans-serif;
+  font-size: ${(props: any) => (props.fontSize ? props.fontSize : "28px")};
+  font-weight: 500;
+  text-align: center;
```
Comment (lipe-dev) [inline lib/components/StatusPercentage/StatusPercentage.styled.ts]: these should be in the theme

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://49--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://49--logrock-pebbles.netlify.app&speed=1

**Review by cld-santos — APPROVED** [review]

**Review by lipe-dev — APPROVED** [review]

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://49--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://49--logrock-pebbles.netlify.app&speed=1

File: lib/components/StatusPercentage/StatusPercentage.tsx (hunk 2 above)
Comment (luizdanielmlima) [reply lib/components/StatusPercentage/StatusPercentage.tsx]: Done, thanks

**Pull request merged**